NDB.Covid19/NDB.Covid19.Droid/Views/Settings/SettingsHowItWorksActivity.cs

```diff
@@ -38,49 +35,44 @@ void Init()
             TextView titleField = FindViewById<TextView>(Resource.Id.settings_how_it_works_title);
 
             titleField.Text = SettingsPage2ViewModel.SETTINGS_PAGE_2_HEADER;
-            textField.TextFormatted = HtmlCompat.FromHtml(SettingsPage2ViewModel.SETTINGS_PAGE_2_CONTENT, HtmlCompat.FromHtmlModeLegacy);
+            textField.TextFormatted = HtmlCompat.FromHtml(SettingsPage2ViewModel.SETTINGS_PAGE_2_CONTENT,
+                HtmlCompat.FromHtmlModeLegacy);
+            LinkUtil.LinkifyTextView(textField);
+            FormatLink(textField);
 
             backButton.Click += new SingleClick((sender, args) => Finish()).Run;
         }
 
-        private void FormatLink (TextView textView) {
-            SpannableStringBuilder spannable = new SpannableStringBuilder (textView.TextFormatted);
-            Object[] spans = spannable.GetSpans (0, spannable.Length (), Class.FromType (typeof (URLSpan)));
-            foreach (URLSpan span in spans) {
-                int start = spannable.GetSpanStart (span);
-                int end = spannable.GetSpanEnd (span);
+        private void FormatLink(TextView textView)
```

Comment:
Add unit tests for this method

Reply:
This is purely Android method. we have no environment for such tests